torch/csrc/jit/fuser/codegen.cpp

```diff
@@ -356,6 +375,10 @@ std::string generateKernel(
       AT_ASSERT(use_cuda);
       has_random = true;
     }
+
+    if (n->mustBeNone()) {
```

Comment:
Can you put this continue logic with the other continue logic above and update the constant?